### DISCUSSION TITLE
Is storing certs on Git secure? What if attackers get hold of certs?

### DISCUSSION THREAD

**Discussion by coderscode2** [fastlane/fastlane#17562] [Q&A] [answered]
We are in the process of migrating from bitrise to circle CI. As a part of this, we need to integrate Fastlane. 
The issue that we see with fast lane is storing the certs/profiles on git. Event though the passphrase sits in the circle CI environment variables, what if attackers get hold off the certs git repo? What would happen if they just have certs but no passphrase? Please let me know the possible disasters that can happen. Thanks

**Comment by rogerluan** [comment, marked as answer]
Hey @coderscode2 👋 

All relevant info can be found here 🤗 https://codesigning.guide

tl;dr: yes, it's secure. 

Good luck!
